src/assets.ts

```diff
@@ -294,7 +292,7 @@ function findExecutableProjectJsonProjects(projects: protocol.DotNetProject[], c
     return result;
 }
 
-function containsDotNetCoreProjects(workspaceInfo: protocol.WorkspaceInformationResponse) {
+export function containsDotNetCoreProjects(workspaceInfo: protocol.WorkspaceInformationResponse) {
```

Comment:
It feels like this helper should move someplace else if it needs to be accessed by more than just the assets generation. Perhaps `src/omnisharp/protocol.ts`? There's already other helpers like this there: https://github.com/OmniSharp/omnisharp-vscode/blob/master/src/omnisharp/protocol.ts#L589-L592.